cpp/src/io/csv/writer_impl.cu

```diff
@@ -146,6 +146,12 @@ struct column_to_strings_fn {
   {
   }
 
+  ~column_to_strings_fn()                                      = default;
+  column_to_strings_fn(column_to_strings_fn const&)            = delete;
+  column_to_strings_fn& operator=(column_to_strings_fn const&) = delete;
+  column_to_strings_fn(column_to_strings_fn&&)                 = delete;
+  column_to_strings_fn& operator=(column_to_strings_fn&&)      = delete;
```

Comment:
Any particular reason we explicitly disable them?

Reply:
The `column_to_strings_fn` is not meant to be copied (https://github.com/rapidsai/cudf/pull/14313#discussion_r1374549183)

Reply:
Good to know! Thanks!